<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="le_username">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logout">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>158</width>
     <height>293</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="btn_selstock">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查看货物库存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_outable">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>进货单管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_incheck">
      <property name="text">
       <string>审核进货单</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_intable">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>出货单管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_outcheck">
      <property name="text">
       <string>审核出货单</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_selneed">
        <property name="text">
         <string>查看缺货单</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_stockmanage">
        <property name="text">
         <string>库存管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_need">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>缺货处理</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
